packages/renderer/src/hooks/haveno/useSetMoneroNodeSettings.ts

```diff
@@ -0,0 +1,46 @@
+// =============================================================================
+//  Copyright 2022 Haveno
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// =============================================================================
+
+import { QueryKeys } from "@constants/query-keys";
+import { useMutation, useQueryClient } from "react-query";
+import { useHavenoClient } from "./useHavenoClient";
+
+interface SetMeneroNodeSettingsQueryData {
```

Comment:
Code convention: we use the pattern `SomethingVariables` or simply, `Variables`. [Here's an example](https://github.com/Viabl-Ventures/haveno-ui/blob/develop/packages/renderer/src/hooks/haveno/useCreateAccount.ts#L20)

Reply:
Fixed.